### ISSUE TITLE
GTK/GDK Applications fail to launch

### ISSUE THREAD

**Issue by jaykijay** [sailfishos-flatpak/main#15]
Every GTK/GDK only application (in this case: com.github.xournalpp.xournalpp) fails with error:
`The Wayland compositor does not provide any supported shell interface, not using Wayland display`
Is it possible to install a supported shell interface like xdg-shell?

**Comment by rinigus** [comment]
Yes, it is known limitation due to the use of QtWayland 5.2. In theory, it is possible to use some other Wayland compositor inside Flatpak Runner and get xdg-shell extension through that. I am/was hoping to get Qt update which would allow us to use QtWayland 5.12 which would bring the corresponding compatibility. Probably more than a year to wait, though.

If someone would like to hook some other Wayland composer and use it for xdg-shell compatibility, PRs are welcome.
